src/content/en/updates/posts/2016/05/houdini.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: updates/post
+title: "Houdini – Demystifying CSS"
+description: "Houdini is a collection of APIs that expose the CSS engine’s internals to developers"
+published_on: 2016-05-20
+updated_on: 2016-05-20
+authors:
+  - surma
+tags:
+  - houdini
+  - css
+featured_image: /web/updates/images/2016/05/houdini/compworklet_small.png
+
+---
+
+Have you ever thought about the amount of work CSS does? You change a single attribute and suddenly your entire website appears in a different layout. It’s kind of *magic* in that regard (can you tell where I am going with this?!). So far, we – the community of web developers – have only been able to witness and observe the magic. What if we want to come up with our own magic tricks and perform them? What if we want to *become the magician*? Enter Houdini!
```

Comment:
Generally we want line breaks around 80 characters. This will become more important as we move into DevSite.